config/config.py

```diff
@@ -56,6 +57,41 @@ def build_config():
             "tau_ES_shift_DM11": 1.0,
         },
         "mt": {
+            "singlemoun_trigger": [
```

Comment:
you could add an era dependency here: "singlemuon_trigger": {"ERA_201...": [...]} and below

Reply:
Updated